data/json/mapgen/animalshelter.json

```diff
@@ -0,0 +1,271 @@
+[
+  {
+    "id": "animalshelter",
+    "type": "vehicle_group",
+    "vehicles": [
+      [ "animalctrl", 300 ],
+      [ "cube_van", 100 ],
+      [ "cube_van_cheap", 50 ]
+    ]
+  },
+  {
+    "id": "animalshelter_softdrug",
+    "type": "item_group",
+    "items": [
+      [ "bandages", 50 ],
+      [ "saline", 20 ],
+      [ "vitamins", 15 ],
+      [ "calcium_tablet", 15 ],
+      [ "nyquil", 5 ],
+      [ "disinfectant", 15 ],
+      [ "elec_hairtrimmer", 15 ],
+      [ "survnote", 5 ],
```

Comment:
Also curious on why you added "survivor's note" to item_group "animalshelter_softdrug". These survivor's notes have various flavor texts on them, many of which I'd imagine being out of place finding them with various medical supplies in an animal shelter.